spec/change_set_persisters/change_set_persister_spec.rb

```diff
@@ -644,8 +644,8 @@
         expect(members.first).to be_kind_of FileSet
 
         expect(members.first.date_of_digitization).not_to be_empty
-        expect(members.first.date_of_digitization.first).to be_a Time
-        expect(members.first.date_of_digitization.first).to eq DateTime.iso8601("2009-03-30T19:49:13.000Z").to_time.utc
+        expect(members.first.date_of_digitization.first).to be_a DateTime
+        expect(members.first.date_of_digitization.first).to eq DateTime.iso8601("2009-03-30T19:49:13.000Z").utc
```

Comment:
We won't need any data migration or anything?

Reply:
We shouldn't. This was just a parsing bug in Valkyrie that's gotten fixed, so it'll parse out as a DateTime now and render back the same way.